octoprint_mrbeam/templates/tab_designstore.jinja2

```diff
@@ -1,42 +1,41 @@
 
 <div class="tab-pane" id="designstore">
-    <iframe id="design_store_iframe" scrolling="no" frameBorder="0" style="width: 100%; height: 90vh; overflow: hidden; display: none"></iframe>
+    <iframe id="design_store_iframe" scrolling="no" frameBorder="0"></iframe>
     <div id="design_store_offline_placeholder">
 {#        Seems like you are not connected to the internet. If you want to use the Design Store, please connect to the internet and reload the page.#}
 
-        <div class="row-fluid no-internet-connection-section"
-         style="position: absolute; top: 50%; transform: translateY(-55%); display: block;">
-        <div class="span12">
-            <div class="row-fluid">
-                <div class="span3">
-                </div>
-                <div class="span6">
-                    <div class="row-fluid">
-                        <div class="span3">
-                            <div class="no-internet-icon">
-                                <img src="/plugin/mrbeam/static/img/no_internet.svg" style="position: absolute; top: 50%; transform: translateY(-80%); padding: 0 5%;">
-                            </div>
-                        </div>
-                        <div class="span9">
-                            <div class="no-internet-content" style="text-align: center; font-size: 20px; padding: 0 60px;">
-                                <h4 style="font-size: 20px; margin-bottom: 40px;">{{ _('THE MR BEAM DESIGN STORE IS OFFLINE') }}</h4>
-                                <p style="margin-bottom: 30px;">{{ _('In order to use our store it is necessary that your Mr Beam is connected to the internet.') }}</p>
-                                <p style="margin-bottom: 75px;">{{ _('Please connect your Mr Beam to your router with a cable, or via WiFi.') }}</p>
-                                <div>
-                                    <button class="btn refresh-connection" onClick="window.location.reload();">
-                                        {{ _('Refresh') }}
-                                    </button>
+        <div class="row-fluid no-internet-connection-section">
+            <div class="span12">
+                <div class="row-fluid">
+                    <div class="span3">
+                    </div>
+                    <div class="span6">
+                        <div class="row-fluid">
+                            <div class="span3">
+                                <div class="no-internet-icon">
+                                    <img src="/plugin/mrbeam/static/img/no_internet.svg">
                                 </div>
+                            </div>
+                            <div class="span9">
+                                <div class="no-internet-content">
+                                    <h4>{{ _('The Mr&nbsp;Beam Design Store is not reachable / available / accessible') }}</h4>
```

Comment:
I think this should only have one of the three words: reachable, available or accesible...